### DISCUSSION TITLE
How to Make MAME Automatically Pause When Inactive?

### DISCUSSION THREAD

**Discussion by staaandup** [mamedev/discussions#113] [Q&A]
Hi,
I noticed that MAME doesn't pause when the window is inactive. Is there a way to make it pause automatically when I switch to another window?

**Comment by cuavas** [comment]
There isn’t.

**Comment by staaandup** [comment]
@cuavas Can you add it?

**Reply by cuavas** [reply]
It isn’t straightforward.  The biggest obstacle is that the interface between the OS-specific window handling code and the emulation is rubbish, and fixing it is a lot of work (same as why mouse capture behaviour still sucks, you get multiple mouse pointers in MAME windows, etc.).  Interaction with the multi-window debugger also complicates things.
